client/utils/utils.go

```diff
@@ -109,13 +87,12 @@ func CalculateGas(queryFunc func(string, common.HexBytes) ([]byte, error), cdc *
 		return
 	}
 	adjusted = adjustGasEstimate(estimate, adjustment)
-	fmt.Fprintf(os.Stderr, "gas: [estimated = %v] [adjusted = %v]\n", estimate, adjusted)
 	return
 }
 
 func adjustGasEstimate(estimate int64, adjustment float64) int64 {
 	if adjustment == 0 {
-		return int64(DefaultGasAdjustment * float64(estimate))
```

Comment:
Ahhh, yes, I don't think you need this here. `NewCLIContext` could have it set.

Reply:
Done, please review